app/controllers/contact_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
         GroupMailer.anonymous_mail(mail_array, contact_params[:body], contact_params[:title]).deliver_now
       end
 
-     redirect_to :contact_index
+      redirect_to :contact_index, flash: {notice: "your mail has been sent to " + build_contacts_string(contact_params[:to_whom])}
```

Comment:
A final comment: the text should be localized so that we can have an English version and a Swedish version.